GNUmakefile

```diff
@@ -1,38 +1,41 @@
-CC?=cc
```

Comment:
GNU Make always sets CC to either `cc` or the vendor/distribution compiler preference, so better to not set it here.